src/registrar/tests/test_views.py

```diff
@@ -137,11 +137,31 @@ def setUp(self):
         self.app.set_user(self.user.username)
         self.TITLES = ApplicationWizard.TITLES
 
+    def test_application_form_intro_acknowledgement(self):
+        """Tests that user is presented with intro acknowledgement page"""
+        intro_page = self.app.get(reverse("application:"))
+        self.assertContains(intro_page, "You're about to start your .gov domain request")
+
```

Comment:
Wrote this test asserting that when you access an application through clicking 'Edit', you bypass the intro page. It works:

```
def test_application_form_intro_is_skipped_when_edit_access(self):
        """Tests that user is NOT presented with intro acknowledgement page when accessed through 'edit'"""        
        completed_application(status=DomainApplication.ApplicationStatus.STARTED, user=self.user)
        home_page = self.app.get("/")
        self.assertContains(home_page, "city.gov")
        # click the "Edit" link
        detail_page = home_page.click("Edit", index=0)
        # Check that the response is a redirect
        self.assertEqual(detail_page.status_code, 302)
        # You can access the 'Location' header to get the redirect URL
        redirect_url = detail_page.url
        self.assertEqual(redirect_url, "/register/organization_type/")
```

Reply:
Thanks.  I added the test